templates/nginx.conf

```diff
@@ -74,6 +75,13 @@ server {
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header Host $host;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+      # Response cache for unauthenticated users
+      proxy_cache lemmy_cache_{{domain}};
+      proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
+      proxy_cache_valid 1m;
```

Comment:
I'd be up for increasing this also.

@Nutomic anything preventing merging this?

Reply:
I deployed it on https://voyager.lemmy.ml/ just now, and it seems to work well. Data is getting cached as expected when user is logged out, and not cached when user is logged in. I also didnt find any leakage of private data.

Though it would be good to update it for using https://github.com/LemmyNet/lemmy-ui/pull/1641 once that is deployed as its a bit cleaner.